tests/Unit/Domain/Test_DomainCreators.cpp

```diff
@@ -224,3 +226,23 @@ SPECTRE_TEST_CASE("Unit.Domain.DomainCreators.TestHelperFunctions",
   tnsr::I<double, 3, Frame::Logical> expected{{{-1.0, -1.0, 1.0}}};
   CHECK(point_in_neighbor_frame<3>(custom_orientation, test_point) == expected);
 }
+
+SPECTRE_TEST_CASE("Unit.Domain.DomainCreators.Disk.Separation",
```

Comment:
1. How does this test that the initial refinement and grid points are set correctly?

2. How does this test that the initial refinement is 2:1 balanced between Elements initially?

Reply:
2. will be tested by a function I am currently implementing, so whoever gets their PR merged second can test this for disk